Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -253,13 +253,13 @@ private Parameter ParameterToSpeckle(DB.Parameter rp, bool isTypeParameter = fal
           if (sp.value == null)
             sp.value = rp.AsValueString();
           break;
-          //case StorageType.ElementId:
-          //  // NOTE: if this collects too much garbage, maybe we can ignore it
-          //  var id = rp.AsElementId();
-          //  var e = Doc.GetElement(id);
-          //  if (e != null && CanConvertToSpeckle(e))
-          //    sp.value = ConvertToSpeckle(e);
-          //  break;
+        case StorageType.ElementId:
```

Comment:
We should keep this commented as otherwise a lot of garbage is collected too

Reply:
ah tru, now that we're no longer using this method in the column conversion, I can safely comment this out again!